xbee_remote_at_block.py

```diff
@@ -50,6 +57,21 @@ def _remote_at(self, command, parameter, dest_addr):
         # frame_id is an arbitrary value, 1 hex byte, used to associate sent 
         # packets with their responses. If set to 0 no response will be sent.
         # Could be a block property.
-        self._xbee.send(
-            'remote_at', frame_id=b'\x01', dest_addr=dest_addr,
-            command=command, parameter=parameter)
+        if self.digimesh():
+            self._xbee.send('remote_at',
+                            id=b'\x17',
+                            frame_id=b'\x01',
+                            dest_addr_long=dest_addr,
+                            reserved=b'\xFF\xFE',
+                            options=b'\x02',
+                            command=command,
+                            parameter=parameter)
+        else:
+            self._xbee.send('remote_at',
+                            id=b'\x17',
+                            frame_id=b'\x01',
+                            dest_addr_long=b'\x00\x00\x00\x00\x00\x00\x00\x00',
```

Comment:
I wanted to be consistent, but you make a good point and a comment is a better solution.